gettsim/tests/test_data/test_dfs_tax_transfer.csv

```diff
@@ -0,0 +1,15 @@
+hid,tu_id,pid,head_tu,hhtyp,m_wage,east,age,selfemployed,haskids,m_self,m_pensions,pkv,m_wage_l1,months_ue,months_ue_l1,months_ue_l2,alg_soep,m_pensions,w_hours,child_num_tu,adult_num_tu,byear,exper,avg_ep,EP,child,m_childcare,m_imputedrent,m_kapinc,m_vermiet,renteneintritt,zveranl,m_pensions,w_hours,ineducation,zveranl,handcap_degree,alleinerz,miete,wohnfl,eigentum,heizkost,child11_num_tu,cnstyr,mietstufe,m_transfers,hh_korr,year
```

Comment:
While the values don't matter much here, they don't make sense for `exper`, `EP` (some are negative)

Reply:
Shouldn't we calculate some of these variables inside anyway? They are in the excel sheet, but is this just for convenience or just general? For example in the sheets 'EP' = 'exper' * 'avg_ep'

Reply:
I don't know, you created that sheet, didn't you? ;)

Reply:
I don't know! :) See above, I'll move that!